### PR TITLE
Token activation balance update

### PR DESCRIPTION
**Pull request by alicjakujawa**
## Description

This PR fixes buttons alignment as well as updating balance after payment action 

**Changes** 🏗

* Updated buttons alignment
* Added refetch query to payment saga

Resolves DEV-230

### CODE REVIEW

**Review by ArmandoGraterol — COMMENTED** [review]
Nice, looks like the improvement is working just fine! I just have a question, should not this be merged on the Token Activation feature branch? Right now it's pointing at master

File: src/modules/users/components/UserTokenActivationButton/UserTokenActivationButton.css [hunk 1]
```diff
@@ -1,23 +1,26 @@
 .tokens {
-  display: flex;
-  justify-content: center;
-  align-items: center;
   margin-right: -3px;
   padding: 0 8px;
   height: 24px;
   position: relative;
-  border: 0;
+  border: 1px solid transparent;
```
[the remark this answers is not in the document]
Comment (alicjakujawa) [reply src/modules/users/components/UserTokenActivationButton/UserTokenActivationButton.css:6]: Better transparent, than you wont have weird "buttons moving"

**Review by ArmandoGraterol — APPROVED** [review]
Looks good to me :ok_hand:

**Comment by alicjakujawa** [comment]
> Looks good! I am not 100% sure but I think it's still misaligned by 1px - i was using the ruler to check. If you give padding-top: 1px to the .tokens class they would be perfectly aligned and it seems it won't screw anything else, but you need to check. But again, I'm not 100% sure.

Screenshot 2021-03-03 at 15 29 46

Is it good now? 🤣

File: src/modules/users/components/UserTokenActivationButton/UserTokenActivationButton.css [hunk 2]
```diff
@@ -1,23 +1,32 @@
 .tokens {
   display: flex;
-  justify-content: center;
   align-items: center;
   margin-right: -3px;
   padding: 0 8px;
```
Comment (rdig) [inline src/modules/users/components/UserTokenActivationButton/UserTokenActivationButton.css:5]: This still doesn't render on the same line :cry: At least for me

Screenshot from 2021-03-03 20-06-46

This is the change that makes it render on the same line for me:
```diff
diff --git a/src/modules/pages/components/RouteLayouts/UserNavigation.css b/src/modules/pages/components/RouteLayouts/UserNavigation.css
index 4e421823d..1f4fb51d6 100644
--- a/src/modules/pages/components/RouteLayouts/UserNavigation.css
+++ b/src/modules/pages/components/RouteLayouts/UserNavigation.css
@@ -98,7 +98,7 @@
 .walletAddress {
   display: flex;
   align-items: center;
-  padding: 0 8px;
+  padding: 2px 8px 0;
 
   /*
    * @NOTE We need to do border sizing shenanigans due to how the border color
diff --git a/src/modules/users/components/UserTokenActivationButton/UserTokenActivationButton.css b/src/modules/users/components/UserTokenActivationButton/UserTokenActivationButton.css
index 6bb2e06a0..bcfa77f52 100644
--- a/src/modules/users/components/UserTokenActivationButton/UserTokenActivationButton.css
+++ b/src/modules/users/components/UserTokenActivationButton/UserTokenActivationButton.css
@@ -2,7 +2,7 @@
   display: flex;
   align-items: center;
   margin-right: -3px;
-  padding: 0 8px;
+  padding: 0 8px 2px;
   height: 24px;
   position: relative;
   border: 1px solid transparent;
```

Please try it on your machine locally and if it works for you too, then add it in

**Review by rdig — CHANGES_REQUESTED** [review]
Still doesn't render on the same line for me :(

I've fixed it locally, please try my changes on your machine.

File: src/modules/users/components/UserTokenActivationButton/UserTokenActivationButton.css (hunk 2 above)
Comment (alicjakujawa) [reply src/modules/users/components/UserTokenActivationButton/UserTokenActivationButton.css]: works fine as well, updated

File: src/modules/pages/components/RouteLayouts/UserNavigation.css [hunk 3]
```diff
@@ -96,7 +96,9 @@
 }
 
 .walletAddress {
-  padding: 0 8px;
+  display: flex;
+  align-items: center;
```
Comment (rdig) [inline src/modules/pages/components/RouteLayouts/UserNavigation.css]: setting this to flex broken the alignment again :(

Remove these two lines and we're all good.

**Review by rdig — APPROVED** [review]
Yep, we're good to go.

I think however that this PR needs to be rebased, as I see a lot of "weird" commits:

Screenshot from 2021-03-09 15-13-53

Please be careful before merging.

**Pull request closed without merging**